packages/web-components/tests/karma.conf.js

```diff
@@ -43,21 +43,25 @@ module.exports = function setupKarma(config) {
 
     browsers: (browsers.length > 0 ? browsers : ['ChromeHeadless']).map(normalizeBrowser),
 
-    frameworks: ['jasmine', 'snapshot'],
+    frameworks: ['jasmine', 'snapshot', 'aChecker'],
 
     client: {
       jasmine: {
         random: !!random,
       },
     },
 
-    files: ['src/polyfills/index.ts', 'tests/utils/snapshot.js', 'tests/snapshots/**/*.md'].concat(
-      specs.length > 0 ? specs : ['tests/karma-test-shim.js']
-    ),
+    files: [
+      'tests/utils/achecker-compliance.js',
+      'tests/a11y/karma-setup-context.js',
+      'src/polyfills/index.ts',
+      'tests/utils/snapshot.js',
+      'tests/snapshots/**/*.md',
+    ].concat(specs.length > 0 ? specs : ['tests/a11y/karma-test-shim.js']),
 
     preprocessors: {
-      'src/**/*.[jt]s': ['webpack', 'sourcemap'], // For generatoring coverage report for untested files
-      'tests/karma-test-shim.js': ['webpack', 'sourcemap'],
+      'src/**/*.[jt]s': ['webpack', 'sourcemap'], // For generating coverage report for untested files
+      'tests/a11y/**/*.js': ['webpack', 'sourcemap'],
```

Comment:
Does this run as part of the regular functional unit tests? Have you noticed any test failures when running?

Reply:
yes it runs with `yarn test:unit`.

and yes lots of failures when runing 

![Screen Shot 2022-01-10 at 12 59 02 PM](https://user-images.githubusercontent.com/20210594/148838556-b75bf11a-f328-4d00-b65c-5535151b3f00.png).

A lot of errors seem to be missing roles 
![Screen Shot 2022-01-10 at 1 00 06 PM](https://user-images.githubusercontent.com/20210594/148838686-9164d87f-d1e8-4f12-8927-a14d83c3ae2e.png)

43 failed, 102 sucess.

Reply:
> yes it runs with `yarn test:unit`.
> 
> and yes lots of failures when runing
> 
> A lot of errors seem to be missing roles ![Screen Shot 2022-01-10 at 1 00 06 PM](https://user-images.githubusercontent.com/20210594/148838686-9164d87f-d1e8-4f12-8927-a14d83c3ae2e.png)
> 
> 43 failed, 102 sucess.

I wonder if we should separate out the functional unit tests from the a11y tests and run as a separate job? We can probably see if we can keep the same config file, but perhaps pass a flag if we can so that it will run the a11y test configs only in that case?

And when we do that, we can pass the tests regardless of status for now (kinda like how we're doing it with React right now) until we fix the failures then we can enforce it when we're ready?

Reply:
Or if there are enough diffs, we can have a separate `karma.conf.js` specifically for `aChecker` for the a11y specific job...